src/main/java/uk/gov/justice/digital/nomis/jpa/repository/OffenderExternalMovementsRepository.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.justice.digital.nomis.jpa.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import uk.gov.justice.digital.nomis.jpa.entity.OffenderExternalMovement;
+
+import java.util.Optional;
+
+@Repository
+public interface OffenderExternalMovementsRepository extends JpaRepository<OffenderExternalMovement, Long> {
```

Comment:
would be good to have a repository test for this change

Reply:
There are no repository tests at all in this project.  I questioned this and after discussion thought the simplicity of this change didn't warrant the amount of work required.

Reply:
Mike Jackson didn't believe in repo tests.  He thought you were just testing the framework